### ISSUE TITLE
Sort zones by trust level instead of alphabetical order, to give firewalld user an intuitive view

### ISSUE THREAD

**Issue by moontide** [firewalld/firewalld#1018]
### What would you like to be added

When reading document of [Predefined Zones](https://firewalld.org/documentation/zone/predefined-zones.html), I saw this:

> These are the zones provided by firewalld **sorted according to the default trust level of the zones** from untrusted to trusted:

I'm wondering, can we have this in `firewall-config` and `firewall-cmd`?

I guess a new zone option, say Trust Level or Custom Sort Field, will be needed to do that. Ideally, the data type of this option is float/double, but integer is also okay. Negative value should be allowed for zones like `drop` and `block`. Maybe `firewall-config` can use this value to render zone with a color or background-color to give user more intuitive view?

### Why is this needed

To give firewalld user an intuitive view when configuring or listing zones

**Comment by erig0** [comment]
Duplicate of #192.

tl;dr zone should have a priority which determines the dispatch order.
